src/Routers/UsersRouter.js

```diff
@@ -289,6 +333,143 @@ export class UsersRouter extends ClassesRouter {
     }
     return Promise.resolve(success);
   }
+  encryptMFAKey(mfa, encryptionKey) {
+    try {
+      if (!encryptionKey) {
+        return mfa;
+      }
+      const algorithm = 'aes-256-gcm';
+      const encryption = crypto
+        .createHash('sha256')
+        .update(String(encryptionKey))
+        .digest('base64')
+        .substr(0, 32);
+      const iv = crypto.randomBytes(16);
+
+      const cipher = crypto.createCipheriv(algorithm, encryption, iv);
+
+      const encryptedResult = Buffer.concat([
+        cipher.update(mfa),
+        cipher.final(),
+        iv,
+        cipher.getAuthTag(),
+      ]);
+      return encryptedResult.toString('base64');
+    } catch (e) {
+      throw new Parse.Error(212, 'Invalid MFA token');
```

Comment:
Requires Parse JS SDK reference for error code; check for possible inconsistency in error message for same error code.